test/Dialect/RTG/IR/basic.mlir

```diff
@@ -188,11 +188,13 @@ rtg.test @tuples() {
   // CHECK-NEXT: [[IDX0:%.+]] = index.constant 0
   // CHECK-NEXT: [[TRUE:%.+]] = index.bool.constant true
   // CHECK-NEXT: [[TUPLE:%.+]] = rtg.tuple_create [[IDX0]], [[TRUE]] : index, i1
-  // CHECK-NEXT: rtg.tuple_extract [[TUPLE]] at 1 : tuple<index, i1>
+  // CHECK-NEXT: rtg.tuple_extract [[TUPLE]] at 1 : !rtg.tuple<index, i1>
+  // CHECK-NEXT: rtg.tuple_create
   %idx0 = index.constant 0
   %true = index.bool.constant true
   %0 = rtg.tuple_create %idx0, %true : index, i1
-  %1 = rtg.tuple_extract %0 at 1 : tuple<index, i1>
+  %1 = rtg.tuple_extract %0 at 1 : !rtg.tuple<index, i1>
+  rtg.tuple_create
```

Comment:
It might be worth adding an empty tuple as an argument to the `rtg.test`, just to make sure that the empty tuple type round-trips. Unless I've overlooked a use of it somewhere 🤔